starboard/nplb/BUILD.gn

```diff
@@ -39,29 +39,11 @@ config("wnos") {
 test("nplb") {
```

Comment:
I think it would be more natural to create `souce_set`s and have the `nplb` target manage which `source_set`s it should depend on for which platform

Reply:
I know! I tried that and changes to that and the deps it basically 
amounted to writing a new file. WDYT, should I do that or just
land this?